### PR TITLE
[EI-7] Update message router sample

### PR DESCRIPTION
**Pull request by erandacr**
## Purpose
This will update message router sample bal file, balx file and readme to be compatible with Ballerina 0.95.5

## Goals
Sample to work properly and provide observations as mentioned in the readme

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: samples/message-router/README.md [hunk 1]
```diff
@@ -113,29 +110,32 @@ service<http> GatewayService {
...
         } else {
-
             json payload = {"Error":"No service found"};
             responseMessage.setJsonPayload(payload);
         }
-        resp.forward(responseMessage);
+
+        http:HttpConnectorError respondError = resp.forward(responseMessage);
+
+        if(respondError != null) {
```
Comment (isudana) [inline samples/message-router/README.md]: formatting issue

File: samples/message-router/services/samples/router/BrowseService.bal [hunk 2]
```diff
@@ -94,7 +95,11 @@ service<http> BrowseService {
         }
 
         resp.setJsonPayload(payload);
-        resp.send();
+        http:HttpConnectorError respondError = resp.send();
+
+        if(respondError != null) {
```
Comment (isudana) [inline samples/message-router/services/samples/router/BrowseService.bal]: formatting issue

File: samples/message-router/services/samples/router/OrderService.bal [hunk 3]
```diff
@@ -19,7 +20,11 @@ service<http> OrderService {
         payload = {"Status":"Order placed successfully", "OrderId" : + orderId  };
 
         resp.setJsonPayload(payload);
-        resp.send();
+        http:HttpConnectorError respondError = resp.send();
+
+        if(respondError != null) {
```
Comment (isudana) [inline samples/message-router/services/samples/router/OrderService.bal]: formatting issue

File: samples/message-router/services/samples/router/PaymentService.bal [hunk 4]
```diff
@@ -21,7 +22,11 @@ service<http> PaymentGatewayService {
         }
         
         resp.setJsonPayload(payload);
-        resp.send();
+        http:HttpConnectorError respondError = resp.send();
+
+        if(respondError != null) {
```
Comment (isudana) [inline samples/message-router/services/samples/router/PaymentService.bal]: formatting issue

File: samples/message-router/services/samples/router/PaymentService.bal [hunk 5]
```diff
@@ -41,7 +46,11 @@ service<http> PaymentGatewayService {
         }
 
         resp.setJsonPayload(payload);
-        resp.send();
+        http:HttpConnectorError respondError = resp.send();
+
+        if(respondError != null) {
```
Comment (isudana) [inline samples/message-router/services/samples/router/PaymentService.bal]: formatting issue

File: samples/message-router/services/samples/router/ShipmentService.bal [hunk 6]
```diff
@@ -27,6 +33,10 @@ service<http> ShipmentService {
         // dummy shipment service
         json payload = {"Status":"Shipment details submitted", "Order ID": orderId};
         resp.setJsonPayload(payload);
-        resp.send();
+        http:HttpConnectorError respondError = resp.send();
+
+        if(respondError != null) {
```
Comment (isudana) [inline samples/message-router/services/samples/router/ShipmentService.bal]: formatting issue

File: samples/message-router/services/samples/router/ShipmentService.bal [hunk 7]
```diff
@@ -14,7 +15,12 @@ service<http> ShipmentService {
         // dummy shipment service
         json payload = {"Status":"Shipment details submitted", "Order ID": orderId};
         resp.setJsonPayload(payload);
-        resp.send();
+
+        http:HttpConnectorError respondError = resp.send();
+
+        if(respondError != null) {
```
Comment (isudana) [inline samples/message-router/services/samples/router/ShipmentService.bal]: formatting issue

File: samples/message-router/README.md (hunk 1 above)
Comment (erandacr) [reply samples/message-router/README.md]: fixed

File: samples/message-router/services/samples/router/BrowseService.bal (hunk 2 above)
Comment (erandacr) [reply samples/message-router/services/samples/router/BrowseService.bal]: fixed

File: samples/message-router/services/samples/router/OrderService.bal (hunk 3 above)
Comment (erandacr) [reply samples/message-router/services/samples/router/OrderService.bal]: fixed

File: samples/message-router/services/samples/router/PaymentService.bal (hunk 4 above)
Comment (erandacr) [reply samples/message-router/services/samples/router/PaymentService.bal]: fixed

File: samples/message-router/services/samples/router/PaymentService.bal (hunk 5 above)
Comment (erandacr) [reply samples/message-router/services/samples/router/PaymentService.bal]: fixed

File: samples/message-router/services/samples/router/ShipmentService.bal (hunk 7 above)
Comment (erandacr) [reply samples/message-router/services/samples/router/ShipmentService.bal]: fixed

File: samples/message-router/services/samples/router/ShipmentService.bal (hunk 6 above)
Comment (erandacr) [reply samples/message-router/services/samples/router/ShipmentService.bal]: fixed

**Pull request merged**